### PR TITLE
Fit and finish run command & query STUD-65529 STUD-65530

### PR DESCRIPTION
**Pull request by IoanDanielSavu**
## Description
Fit and finish for the run command and run query activities

## Jira 
https://uipath.atlassian.net/browse/STUD-65529
https://uipath.atlassian.net/browse/STUD-65530

### CODE REVIEW

File: Activities/Database/UiPath.Database.Activities/NetCore/ViewModels/ExecuteNonQueryViewModel.cs [hunk 1]
```diff
@@ -86,37 +73,36 @@ protected override void InitializeModel()
...
+            Sql.IsRequired = true;
+            Sql.OrderIndex = propertyOrderIndex++;
+            Sql.Widget = new DefaultWidget { Type = ViewModelWidgetType.TextComposer };
 
             Parameters.OrderIndex = propertyOrderIndex++;
             Parameters.Widget = new DefaultWidget { Type = ViewModelWidgetType.Dictionary };
 
-            AffectedRecords.OrderIndex = propertyOrderIndex++;
-            AffectedRecords.Widget = new DefaultWidget { Type = ViewModelWidgetType.Input };
+            TimeoutMS.OrderIndex = propertyOrderIndex;
```
Comment (alexandru-petre) [inline Activities/Database/UiPath.Database.Activities/NetCore/ViewModels/ExecuteNonQueryViewModel.cs:100]:
```suggestion
            TimeoutMS.OrderIndex = propertyOrderIndex++;
```

File: Activities/Database/UiPath.Database.Activities/NetCore/ViewModels/ExecuteQueryViewModel.cs [hunk 2]
```diff
@@ -86,37 +72,36 @@ protected override void InitializeModel()
...
+            Sql.IsRequired = true;
+            Sql.OrderIndex = propertyOrderIndex++;
+            Sql.Widget = new DefaultWidget { Type = ViewModelWidgetType.TextComposer };
 
             Parameters.OrderIndex = propertyOrderIndex++;
             Parameters.Widget = new DefaultWidget { Type = ViewModelWidgetType.Dictionary };
 
-            DataTable.OrderIndex = propertyOrderIndex++;
-            DataTable.Widget = new DefaultWidget { Type = ViewModelWidgetType.Input };
+            TimeoutMS.OrderIndex = propertyOrderIndex;
```
Comment (alexandru-petre) [inline Activities/Database/UiPath.Database.Activities/NetCore/ViewModels/ExecuteQueryViewModel.cs:99]:
```suggestion
            TimeoutMS.OrderIndex = propertyOrderIndex++;
```

File: Activities/Database/UiPath.Database.Activities/DatabaseExecute.cs [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                {
+                    this.parameters = new Dictionary<string, Argument>();
+                }
+                return this.parameters;
+            }
+            set
+            {
+                parameters = value;
+            }
+        }
```
Comment (alexandru-petre) [inline Activities/Database/UiPath.Database.Activities/DatabaseExecute.cs]: I know this is not actually your code, but please let's clean up this property:
- it's by default browsable anyway
- default value is actually new Dictionary
- the backing field can be removed (and its name is not following the convention)

File: Activities/Database/UiPath.Database.Activities/DatabaseExecute.cs [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        [LocalizedDisplayName(nameof(Resources.Activity_DatabaseExecute_Property_ContinueOnError_Name))]
+        [LocalizedDescription(nameof(Resources.Activity_DatabaseExecute_Property_ContinueOnError_Description))]
+        public InArgument<bool> ContinueOnError { get; set; }
+
+        [LocalizedCategory(nameof(Resources.Common))]
+        [LocalizedDisplayName(nameof(Resources.Activity_DatabaseExecute_Property_TimeoutMS_Name))]
+        [LocalizedDescription(nameof(Resources.Activity_DatabaseExecute_Property_TimeoutMS_Description))]
+        public InArgument<int> TimeoutMS { get; set; }
+
+        protected override Task<Action<AsyncCodeActivityContext>> ExecuteAsync(AsyncCodeActivityContext context, CancellationToken cancellationToken)
```
Comment (alexandru-petre) [inline Activities/Database/UiPath.Database.Activities/DatabaseExecute.cs]: We could seal it to avoid being overwritten in the derived classes.

**Review by alexandru-petre — CHANGES_REQUESTED** [review]

File: Activities/Database/UiPath.Database.Activities/NetCore/ViewModels/ExecuteNonQueryViewModel.cs (hunk 1 above)
Comment (IoanDanielSavu) [reply Activities/Database/UiPath.Database.Activities/NetCore/ViewModels/ExecuteNonQueryViewModel.cs:100]: In the spec, `timeout` and `continue on error` properties are on the same line

File: Activities/Database/UiPath.Database.Activities/NetCore/ViewModels/ExecuteQueryViewModel.cs (hunk 2 above)
Comment (IoanDanielSavu) [reply Activities/Database/UiPath.Database.Activities/NetCore/ViewModels/ExecuteQueryViewModel.cs:99]: same as above

**Review by alexandru-petre — APPROVED** [review]

**Pull request merged**